lib/tasks/example_data.rake

```diff
@@ -52,6 +52,8 @@ namespace :example_data do
   task generate: :environment do
     raise "THIS TASK CANNOT BE RUN IN PRODUCTION" if Rails.env.production?
 
+    abort("DB already contains data") if Trainee.any?
```

Comment:
this will produce exit code 1 ? we would want exit code 0 for app to start in this scenario

Reply:
Yes it will you're right. I've updated to use `exit` instead which will return a 0 exit code.